guide/community_vs_core.md

```diff
@@ -0,0 +1,17 @@
+## Community vs Core
+
+To unlock the community’s creative potential we must step aside and decentralize decision making. This should create the space for teams to build the components that they need and help us quickly scale the system. We also need a stable system. With the parts constantly changing it’s difficult to build anything complex. So the solution we have come up with is a hybrid.
+
+- **CORE:** The TDS Core team will look after the most universal and highly governed components in a ‘TDS Core’ repo to ensure a level of stability to the system.
+
+- **COMMUNITY:** The Community will have its own Repo which will have its own Open Governance model. This will give you the flexibility to build and extend our current capabilities as much as you want with lower governance overhead.
+
+- **PRIVATE:** If you would prefer to not share your components yet, you can keep them in your codebase. If at a later date you need to use them elsewhere simple migrate them over.
+
+## Responsibilities Mapped
+![enter image description here](https://lh3.googleusercontent.com/HECfkqC1OF8s8yJ9IGeVYYvrBW4jUuhdKtmEg5lyrO3-d07G58P8AAQrNMjA2o84LoTj5koLv5vKb6E39mjSnID8VCjbJ5Zp61lhst6FYRWIQ15jT0_mlyFKApnbLH2kfnmPBCOOqJudQZlaH54vTT62Xs7OY0lQ0Xp_Qdif27wM0b8MvYk_eWxpg9e0RMx1TipLdQTFxKA-w2HkAUONe04pzckhF8bImu258sWgZMFzcH2654fhuGG7cTkPagoI9loJv4kfAk1nUe41MPLVvoIHaZYODq36p_hHPycMorazC2lYAnazGhdLcYXp3mp4REDlnlRqLYWdT4S6eHYwbh8NgNcJDhjKQFx0BfJxGl_JDvoZCB5-NvXZ5sSRLXmvfXH55qAY7TRAOJURu_y19uPHMCfT_A2f4PZscSvMPFXrrj2S46Y2FqgG1j-xWKesaooTU_MoF2LpMz132ISzLlAAaZHaeWY80hHJ91U6LEt0R0c-YNG1-eqpq7iz9qEuEaVidXaKYGfXFIaY_JFtDa-CwqFSQ6MlJY1sQUT_WbNuqGLhUe7dgLEPFbEl0vBRDWj3wZjg-ECmjztOTIp3K7QyRiRx0FO2M40Gfr3AA8Cr3VGM6wIU68cGPxX6VQ2CLVBBVXddXjnpXZFfd05eEToEOAQHosT3=w1211-h411-no)
+
+  
+
+## How it works
+![enter image description here](https://lh3.googleusercontent.com/nClqb3DXf1oljH4-Nq0XqUr7IDWe5OV0M-kfrZe-KPTqLX_BTeHILuWTRguZtLNFxhHQ82ly0Er3Q9l-AWBQIbY4acaFFO2MVwwF_Zg3GlkJFifLCegr5-_-ZIrlcrT_eok3WciG8GraAuz1HaKcBSbzDxRmnPpM1U1nW9HwzMWA0Njny1RYVG8aKMyzkXJg2Jt0fOt1Aq5INFemGi6hpQuW8_YClU8VxOScC0QycyPl-EMYPGM4uze8nUXZmEf4gPpi16VdZBq7CD2K_Hns4jf7ymX-DD3qi9uoiO3ux6jsYkyPcWwCpbRgt_LGKILOMSkpxTqDoBZTXdKd3nl773gWn5ZXnMKqNXOv7p3fO5l6kpcmP7XiMjrPNhuhKM2lIgq7e2G_ApFLmDlxB2mQrl5y867kNbGTl1CuKSRzGfKfDQhXwsrAasRU13MygHSG0EvImVJKvTu9WhE1bQEDmAOJE7x6x_32gkEpJrni4JT2WBXJ4geIXO3DNhqQYgyMGTLaMSpWNRJ34zLBu0i_0gkv6o7VKjE5W42Ct3g02RgTT-RJyd7f430tLsnCVIAh1b_Aj-xCFshqtTa9yQRrWIiv-igGyATNLYU5GJU=w998-h631-no)
```

Comment:
2nd image does not seem to render when previewing this Markdown file on GitHub, 1st image renders.
<img width="1245" alt="screen shot 2018-06-13 at 8 22 56 am" src="https://user-images.githubusercontent.com/5464587/41351171-94b4b056-6ee3-11e8-815c-9cc5d7a6b021.png">

Reply:
Hey @jackreeves just wondering if you've got bandwidth to resolve the feedback above? Or should we close this PR?